packages/css/src/grid/grid-cell.scss

```diff
@@ -5,14 +5,20 @@
 
 @import "../../utils/breakpoint";
 
-@media screen and (width > $amsterdam-breakpoint) {
-  @for $i from 1 through 12 {
-    .amsterdam-grid-column-start-#{$i} {
-      grid-column-start: $i;
-    }
+.amsterdam-grid-cell:first-child {
+  grid-column-start: content;
+}
+
+.amsterdam-grid-cell.amsterdam-grid-cell-full-width {
+  grid-column: full;
+}
+
+@for $i from 1 through 12 {
+  .amsterdam-grid-cell.amsterdam-grid-column-span-#{$i} {
+    grid-column-end: span $i;
+  }
 
-    .amsterdam-grid-column-span-#{$i} {
-      grid-column-end: span $i;
-    }
+  .amsterdam-grid-cell.amsterdam-grid-column-start-#{$i} {
+    grid-column-start: $i content;
```

Comment:
Adding `content` behind the column number counts only columns with this name, as specified in the `grid-template-columns` of `page-grid`.